pkgs/applications/virtualization/qemu/default.nix

```diff
@@ -3,20 +3,20 @@
 , bison, lzo, snappy, libaio, libtasn1, gnutls, nettle, curl, dtc, ninja, meson
 , sigtool
 , makeWrapper, removeReferencesTo
-, attr, libcap, libcap_ng, socat, libslirp
+, attr, libcap, libcap_ng, socat, libslirp, audit
 , CoreServices, Cocoa, Hypervisor, Kernel, rez, setfile, vmnet
 , guestAgentSupport ? (with stdenv.hostPlatform; isLinux || isNetBSD || isOpenBSD || isSunOS || isWindows) && !minimal
 , numaSupport ? stdenv.isLinux && !stdenv.isAarch32 && !minimal, numactl
 , seccompSupport ? stdenv.isLinux && !minimal, libseccomp
 , alsaSupport ? lib.hasSuffix "linux" stdenv.hostPlatform.system && !nixosTestRunner && !minimal
-, pulseSupport ? !stdenv.isDarwin && !nixosTestRunner && !minimal, libpulseaudio
-, pipewireSupport ? !stdenv.isDarwin && !nixosTestRunner && !minimal, pipewire
-, sdlSupport ? !stdenv.isDarwin && !nixosTestRunner && !minimal, SDL2, SDL2_image
-, jackSupport ? !stdenv.isDarwin && !nixosTestRunner && !minimal, libjack2
-, gtkSupport ? !stdenv.isDarwin && !xenSupport && !nixosTestRunner && !minimal, gtk3, gettext, vte, wrapGAppsHook3
+, pulseSupport ? !stdenv.isDarwin && !nixosTestRunner && !minimal && lib.meta.availableOn stdenv.hostPlatform libpulseaudio, libpulseaudio
+, pipewireSupport ? !stdenv.isDarwin && !nixosTestRunner && !minimal && !stdenv.hostPlatform.isStatic, pipewire
```

Comment:
These should all use meta.availableOn rather than hardcoding the condition.

Reply:
I did not use it for pipewire because the issue is with a transitive dependency (systemd iirc)

Reply:
Pipewire can't be built statically, even if you can build all the dependencies (I've tried) so it would make sense for it to be marked unsupported directly.